Automattic/jetpack
# Social: Remove twitter logo from dashboard

Fixes #31753

## Proposed changes:

* Changed the illustration to be the new one

### Other information:


## Jetpack product discussion


https://github.com/orgs/Automattic/projects/733/views/2?pane=issue&itemId=46658539
## Does this pull request change what data or activity we track or use?


No.
## Testing instructions:






* Create a new JN site with this branch
* Go to the Social admin page
* You should see the correct illustration:

![CleanShot 2023-12-05 at 16 43 12 png](https://github.com/Automattic/jetpack/assets/36671565/9304f82b-41e3-4025-928d-7622c355b0cb)
